packages/@aws-cdk/aws-batch/lib/job-queue.ts

```diff
@@ -106,24 +105,20 @@ export class JobQueue extends Resource implements IJobQueue {
   public readonly jobQueueArn: string;
   public readonly jobQueueName: string;
 
-  constructor(scope: Construct, id: string, props: JobQueueProps = {}) {
+  constructor(scope: Construct, id: string, props: JobQueueProps) {
     super(scope, id, {
       physicalName: props.jobQueueName,
     });
 
+    if (props.computeEnvironments.length === 0) {
+      throw new Error('It\'s not allowed to pass an empty array of JobQueueComputeEnvironment elements.');
```

Comment:
`array` and `JobQueueComputeEnvironment` is redundant here

```suggestion
`computeEnvironments` must be non-empty
```

Reply:
Done!